src/Statistics/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2022 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+cmake_minimum_required(VERSION 3.15)
+
+add_library(Statistics STATIC)
+
+target_sources(Statistics PUBLIC
+                include/Statistics/BinomialConfidenceInterval.h)
+
+target_include_directories(Statistics PUBLIC
+               ${CMAKE_CURRENT_LIST_DIR}/include)
+
+target_sources(Statistics PRIVATE
+                BinomialConfidenceInterval.cpp)
+
+add_executable(WilsonBinomialConfidenceIntervalEstimatorTests)
+
+target_sources(WilsonBinomialConfidenceIntervalEstimatorTests PRIVATE
+          WilsonBinomialConfidenceIntervalEstimatorTests.cpp)
```

Comment:
If I remember correctly the name of the .cpp file for tests should end with "Test" and not with "Tests" so that clang-format realizes it's a test for the file without the "Test" suffix. Either way, we use "Test" in the codebase.